src/Core/OrganizationFeatures/OrganizationDomains/VerifyOrganizationDomainCommand.cs

```diff
@@ -61,6 +66,9 @@ public async Task<OrganizationDomain> VerifyOrganizationDomain(Guid id)
             _logger.LogError("Error verifying Organization domain.", e);
         }
 
+        domain.SetLastCheckedDate();
+        await _organizationDomainRepository.ReplaceAsync(domain);
+
         await _eventService.LogOrganizationDomainEventAsync(domain, EventType.OrganizationDomain_NotVerified);
```

Comment:
The LogOrganizationDomainEventAsyn can be updated with this

```
await _eventService.LogOrganizationDomainEventAsync(domain,
            orgDomain.VerifiedDate != null ? EventType.OrganizationDomain_Verified : EventType.OrganizationDomain_NotVerified);
```

Reply:
Great catch!